src/Controller/CourseController.php

```diff
@@ -48,21 +66,43 @@ public function setCourse(Request $request)
             return new JsonResponse([
                  'error_message' => $errors
              ], Response::HTTP_BAD_REQUEST);
+
         }
 
         $courseUser = new CourseUser();
         $courseUser->setUser($this->getUser());
         $courseUser->setRole(RoleInterface::ADMIN);
         $courseUser->setStatus(StatusInterface::ACTIVE);
 
+        if($course->getIsSubmittable()){
+            $entryTask = new EntryTask();
```

Comment:
Čia nesupratau, kodėl taip kūri, formos yra nestable, gali susikurti `EntryTaskType`, kuris turi visą šitą info ir `CourceType` sukurti `EntryTaskType` field'ą ir submitinant vieną kartą formą iš karto bus automatiškai sukurti visi vidiniai objektai